src/data/full-quiz.js

```diff
@@ -1,4 +1,47 @@
 const fullQuiz = [
+  
+    {
+    Question: "With SQL, how do you select all the records from a table named 'Persons' where the value of the column 'FirstName' starts with an 'a'?",
+    Answer: "SELECT * FROM Persons WHERE FirstName LIKE ‘a%’",
+    Distractor1: "SELECT * FROM Persons WHERE FirstName=’a’",
+    Distractor2: " SELECT * FROM Persons WHERE FirstName=’%a%’",
+    Distractor3: "SELECT * FROM Persons WHERE FirstName=_a",
+    Explanation: "The SQL LIKE clause is used to compare a value to similar values using wildcard operators.",
+    Link: "https://www.freecodecamp.org/news/sql-like-statement-how-to-query-sql-with-wildcard/",
+   },
+  
+  {
+    Question: "Which SQL statement is used to insert new data in a database?",
+    Answer: "INSERT INTO",
+    Distractor1: "ADD NEW",
+    Distractor2: "ADD RECORD",
+    Distractor3: "INSERT NEW",
+    Explanation: "To insert a new row or data into table in a database INSERT INTO statement is used",

```

Comment:
I think we can simplify the explanation.

Proposed fix:
If you want to add data to your SQL table, then you can use the INSERT INTO statement.

Reply:
```suggestion
    Explanation: "If you want to add data to your SQL table, then you can use the INSERT INTO statement.",
```